cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -904,6 +906,21 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 			logWithId.Printf("[container] 🎉 Image uploaded (%s)!", digest.String())
 			targetResult.Options = &target.ContainerTargetResultOptions{URL: client.Target.String(), Digest: digest.String()}
 
+		case *target.PulpOSTreeTargetOptions:
+			targetResult = target.NewPulpOSTreeTargetResult(nil)
+			archivePath := filepath.Join(outputDirectory, jobTarget.OsbuildArtifact.ExportName, jobTarget.OsbuildArtifact.ExportFilename)
+
+			creds := pulp.Credentials{
+				Username: targetOptions.Username,
+				Password: targetOptions.Password,
+			}
```

Comment:
We tend to pass credentials in the target options only for the on-premise use case. Is there some plan to allow configuring some credentials for Pulp on workers? I mean, we may not want to be passing secrets from `image-builder` to composer with each request... :shrug:

Reply:
Good point.  I didn't think about the service config here much.  I'll fix it.